content/docs/lists-and-keys.md

```diff
@@ -296,6 +297,7 @@ function NumberList(props) {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/BLvYrB?editors=0010)
+[**Փորձել CodePen-ում**](https://codepen.io/gaearon/pen/BLvYrB?editors=0010)
 
-Sometimes this results in clearer code, but this style can also be abused. Like in JavaScript, it is up to you to decide whether it is worth extracting a variable for readability. Keep in mind that if the `map()` body is too nested, it might be a good time to [extract a component](/docs/components-and-props.html#extracting-components).
+
+Երբեմն, որպես արդյունք, ունենում ենք ավելի մաքուր կոդ, սակայն այս ոճը կարող է չարաշահվել։ Ինչպես JavaScript-ում, ըստ ձեր հայացողության, դուք եք որոշում արժե տարանջատել փոփոխականը ավելի ընթերնելի դարձնելու համար, թե ոչ։ Մտապահեք այն, որ եթե `map()`-ի մարմինը բավականին ներդրված է, ապա դա [կոմպոնենտը տարանջատելու](/docs/components-and-props.html#extracting-components) շատ լավ ժամանակ է։
```

Comment:
Ինչպես JavaScript-ում, ըստ ձեր հայացողության, դուք եք որոշում արժե տարանջատել փոփոխականը ավելի ընթերնելի դարձնելու համար, թե ոչ։ -> Ինչպես JavaScript-ում, ձեր հաեցողության տակ է, որոշել տարանջատել փոփոխականը ավելի ընթերնելի դարձնելու համար, թե ոչ։

Reply:
Մտապահեք այն, որ եթե `map()`-ի մարմինը բավականին ներդրված է, ապա դա [կոմպոնենտը տարանջատելու](/docs/components-and-props.html#extracting-components) շատ լավ ժամանակ է։ -> Մտապահեք. եթե `map()`-ի մարմինը բավականին ներդրված է, ապա միգուցե կարիք կա [կոմպոնենտը տարանջատելու](/docs/components-and-props.html#extracting-components)։